package.json

```diff
@@ -37,7 +37,7 @@
   },
   "scripts": {
     "flow": "flow",
-    "prettier": "prettier --write 'lib/**/*.js' 'spec/**/*.js'",
+    "prettier": "prettier --write lib/**/*.js spec/**/*.js",
```

Comment:
I think the single vs double quotes issue might be platform specific or maybe an npm version issue?

Some sort of quotes are required for the glob to work properly (on Ubuntu at least). 

Maybe try escaped double quotes?:
` "prettier": "prettier --write \"lib/**/*.js\" \"spec/**/*.js\"",`

Reply:
Ah sorry for my hasty conclusion.

I confirmed escaped quotes work on Windows10 with npm version 6.4.1.
Shall I commit this or leave this as it was and close this PR ?